### PR TITLE
Adding export contact list content to Automation docs

### PR DESCRIPTION
**Pull request by Whatthefoxsays**
**Description of the change**:
**Reason for the change**:
**Link to original source**:

Closes #

### CODE REVIEW

File: content/docs/ui/sending-email/getting-started-with-automation.md [hunk 1]
```diff
@@ -164,15 +164,17 @@ If you keep the entry criteria as "All Contacts" it's important to remember that
 
 1. Repeat steps 5-16 from creating a custom automation.
 
-## Manage Contacts
+## Add contacts
+
+Adding contacts to a list that's designated as the entry criteria for an automation will trigger that automation. The Automation beta experience offers several ways to add contacts to lists, including CSV upload and signup forms.
```
Comment (mollyfriederich) [inline content/docs/ui/sending-email/getting-started-with-automation.md]:
```suggestion
Adding contacts to a list that's designated as the entry criteria for an automation will trigger that automation. The Automation beta experience offers several ways to add contacts to lists, including CSV upload, using signup forms, or via the Contact Management APIs. 
```

Comment (mollyfriederich) [reply content/docs/ui/sending-email/getting-started-with-automation.md]: Not sure if we can link "Contact Management APIs" to the section below?

File: content/docs/ui/sending-email/getting-started-with-automation.md [hunk 2]
```diff
@@ -241,13 +242,31 @@ To make your form accessible to new contacts, you can either send them a link ho
...
+<call-out type="warning">
+
+Exported CSV files are only available for download for 72 hours after the export is initialized. 
+
+</call-out>
 
-### Edit or Remove Contacts
+## Edit or Remove Contacts
+
+During this early phase of beta, you can only edit contact details from the All Contacts page.
```
Comment (mollyfriederich) [inline content/docs/ui/sending-email/getting-started-with-automation.md]:
```suggestion
During this early phase of beta, you can only view and edit contact details from the All Contacts page. In the future you'll be able to view and edit contact details from lists. You'll also be able to delete contacts. 
```

**Comment by thinkingserious** [comment]
Hello @Whatthefoxsays,<br /><br/>Thanks again for the PR!<br />                                              <br />We appreciate your contribution and look forward to                                              continued collaboration. Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**